kieserver/README.adoc

```diff
@@ -0,0 +1,8 @@
+# Red Hat Decision Manager 7 - Decision Central OpenShift container image
+
+NOTE: Extends link:https://github.com/jboss-container-images/rhdm-7-image/kieserver[Red Hat Decision Manager Execution Server 7.0 container image]
```

Comment:
The URL `https://github.com/jboss-container-images/rhdm-7-image/kieserver` is not valid. Not sure what to change it to though - either just the base repo URL or link specifically to the branch holding the content.

Reply:
It will be available when the base image gets merged.

Reply:
I am afraid that's not the case - the standalone image PR was merged, but the URL is not valid.

You would need something like https://github.com/jboss-container-images/rhdm-7-image/tree/rhdm70-dev/kieserver